src/Promitor.Core.Scraping/AzureMonitorScraper.cs

```diff
@@ -107,10 +108,16 @@ protected virtual string DetermineMetricFilter(string metricName, TResourceDefin
         /// </summary>
         /// <param name="metricName">Name of the metric being queried</param>
         /// <param name="resourceDefinition">Contains the resource cast to the specific resource type.</param>
-        /// <param name="dimension">Provides information concerning the configured metric dimension.</param>
-        protected virtual string DetermineMetricDimension(string metricName, TResourceDefinition resourceDefinition, MetricDimension dimension)
+        /// <param name="configuration"></param>
+        protected virtual List<string> DetermineMetricDimensions(string metricName, TResourceDefinition resourceDefinition, AzureMetricConfiguration configuration)
```

Comment:
We should take the union, not one or the other

Reply:
Because they might forget to remove that field :)

but it's OK for me to keep as is